configure

```diff
@@ -1799,6 +1804,11 @@ main()
 	os_vers=$(uname -r)
 	echo "${script_name}: detected ${os_name} kernel version ${os_vers}."
 
+    is_win=0
+    if  [[ $os_name == MSYS* ]] || [[ $os_name == MINGW* ]]  || [[ $os_name == CYGWIN* ]] ;
+    then
+        is_win=1
+    fi
```

Comment:
I have another request. Can you change the nature of `is_win` in `configure` to `"yes"`/`"no"` (instead of `0`/`1`) and then adjust the branches on `$(IS_WIN)` in `common.mk` and `Makefile` accordingly. (I prefer that its potential values match the other "boolean" variables set in `config.mk`, which are all `"yes"`/`"no"` variables.)